app/components/nav.jsx

```diff
@@ -2,23 +2,19 @@ var React = require('react')
 
 let Nav = React.createClass({
 
-  getInitialState: function (props) {
+  getInitialState: function () {
     return {
-      isActiveLiNo: 6
     };
   },
 
-  clickHandler: function(i){
-    this.setState({ isActiveLiNo: i })
-    // console.log(this.state.isActiveLiNo);
-    this.render();
-  },
-
   getLi: function () {
+    // console.log("props",this.props.highlight);
+
+    // make individual li component for ul
     return this.props.components.map( (comp,i) => {
-      return <li id={comp} key={i} className={ this.state.isActiveLiNo === i ? "active" : "" } 
-              onClick={ () => this.clickHandler(i) }>
-                <a href="#">{comp}</a>
+      return <li id={comp} data-key={i} className={ this.props.highlight === i ? "active" : "" } 
```

Comment:
Ah right, you may as well take it out if you're not using it for anything in particular.

If react gives an error about a Key being needed, change it back to `key=`